app/pages/application-details.tsx

```diff
@@ -51,6 +55,8 @@ class ApplicationDetails extends Component<Props> {
           applicationId={this.props.router.query.applicationId}
         />
         <hr />
+
+        <ApplicationWizardConfirmation is_analyst query={query} />
```

Comment:
This component should probably be renamed to something like `ApplicationDetails`